<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.1.3 (Python 3.10.2 on win32)" generated="20220125 19:49:37.458" rpa="false" schemaversion="2">
<suite id="s1" name="Test" source="C:\Users\luidi.oliveira\Desktop\curso\exemplo1\test.robot">
<test id="s1-t1" name="Caso de teste 01: acessar o navegador">
<kw name="Abrir Navegador" library="resource">
<kw name="Open Browser">
<arg>about:blank</arg>
<msg timestamp="20220125 19:49:37.498" level="FAIL">No keyword with name 'Open Browser' found.</msg>
<status status="FAIL" starttime="20220125 19:49:37.498" endtime="20220125 19:49:37.498"/>
</kw>
<status status="FAIL" starttime="20220125 19:49:37.497" endtime="20220125 19:49:37.499"/>
</kw>
<status status="FAIL" starttime="20220125 19:49:37.497" endtime="20220125 19:49:37.500">No keyword with name 'Open Browser' found.</status>
</test>
<status status="FAIL" starttime="20220125 19:49:37.465" endtime="20220125 19:49:37.500"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Test">Test</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20220125 19:49:37.496" level="ERROR">Error in file 'C:\Users\luidi.oliveira\Desktop\curso\exemplo1\resource.robot' on line 2: Importing library 'SeleniumLibrary' failed: ModuleNotFoundError: No module named 'SeleniumLibrary'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\luidi.oliveira\AppData\Local\Programs\Python\Python310\Scripts\robot.exe
  C:\Users\luidi.oliveira\AppData\Local\Programs\Python\Python310\python310.zip
  C:\Users\luidi.oliveira\AppData\Local\Programs\Python\Python310\DLLs
  C:\Users\luidi.oliveira\AppData\Local\Programs\Python\Python310\lib
  C:\Users\luidi.oliveira\AppData\Local\Programs\Python\Python310
  C:\Users\luidi.oliveira\AppData\Local\Programs\Python\Python310\lib\site-packages</msg>
</errors>
</robot>
